### ISSUE TITLE
Can not find some class which exist when using r2 to analyze apk.

### ISSUE THREAD

**Issue by BurningTeng** [radareorg/radare2#18003]
# This template is meant for bug reports, if you have a feature request, please be as descriptive as possible and delete the template

Make sure you are testing using the latest git version of radare2 before submitting any issue.

*If you would like to report a bug, please fill the template below*

### Work environment

| Questions                                            | Answers
|------------------------------------------------------|--------------------
| OS/arch/bits (mandatory)                             | Ubuntu20.04
| File format of the file you reverse (mandatory)      | apk
| Architecture/bits of the file (mandatory)            | dex
| r2 -v full output, **not truncated** (mandatory)     | radare2 4.6.0-git 25326 @ linux-x86-64 git.4.4.0-989-ged8e66a17 commit: ed8e66a171ce63693865b67855ee82a6e15a8c01 build: 2020-12-06__19:43:55

### Expected behavior
I can find all classes contains in APK
### Actual behavior
I can not find all classes contains in APK
### Steps to reproduce the behavior 
- r2 apk://tb.apk
- Use command `icq~MethodEnum`
- There is nothing output.

When I use apktool to unzip apk, the class can be found.
![burning](https://user-images.githubusercontent.com/20567891/101351217-398d2500-38cb-11eb-8c33-25be3fbb04b8.png)

Drag and drop zip archives containing the Additional info here, don't use external services or link.

**Comment by BurningTeng** [comment]
The apk is bigger than limitation. You can get it at http://www.hacksprit.top:8090/files/tb.apk

**Comment by trufae** [comment]
I'll tell you next week.

![Screenshot 2020-12-07 at 13 50 40](https://user-images.githubusercontent.com/6431515/101353009-3e35e700-3893-11eb-8f78-0bcc0f7d3513.png)

but my blind guess is that you are greping for a class that is defined in a different .dex file. apktool merges them all when processing them. r2 needs bin.libs=true to do that. you can do this: `grep MethodEnum *.dex`

**Comment by BurningTeng** [comment]
Yes. Defined in diffrent class. But "bin.libs=true" seems does not work.
```
[0x001e9598]> set bin.libs=true
[0x001e9598]> aaaa
[x] Analyze all flags starting with sym. and entry0 (aa)
[x] Analyze function calls (aac)
[x] Analyze len bytes of instructions for references (aar)
[x] Check for vtables
[x] Finding xrefs in noncode section with anal.in=io.maps
[x] Analyze value pointers (aav)
[x] Value from 0x0014b0bc to 0x003b05b2 (aav)
[x] 0x0014b0bc-0x003b05b2 in 0x14b0bc-0x3b05b2 (aav)
[0x0015fb0c esil_neg: unknown reg v39d references (aaef)
0x0015fa86 esil_neg: unknown reg v36
0x0015f9da esil_neg: unknown reg v39
0x0015f6dc esil_neg: unknown reg v40
0x001622fa esil_neg: unknown reg v37
0x001627a6 esil_neg: unknown reg v37
0x002e2120 esil_neg: unknown reg v36
0x0035a4c8 esil_neg: unknown reg v36
[x] Emulate functions to find computed references (aaef)
[x] Type matching analysis for all functions (aaft)
[x] Propagate noreturn information
[x] Use -AA or aaaa to perform additional experimental analysis.
[x] Finding function preludes
[x] Enable constraint types analysis for variables
[0x001e9598]> grep MethodEnum *.dex
[0x001e9598]> !grep MethodEnum *.dex
[0x001e9598]> !grep MethodEnum *.dex
[0x001e9598]> !grep MethodEnum *.dex
[0x001e9598]> !grep MethodEnum  *.dex
[0x001e9598]> grep MethodEnum  *.dex

```

**Comment by trufae** [comment]
bin.libs only works when you open a binary. if the binary is already opened it wont do anything until tyou open another binary. so you have to set it with `r2 -e bin.libs=true ...`

**Comment by trufae** [comment]
also its expected that `!grep` wont work with the files INSIDE the zip because you are opening the zip with r2, not with the shell.

**Comment by BurningTeng** [comment]
Thanks your explanation. But `r2 -e bin.libs=true apk://tb.apk` does not work.
![burning](https://user-images.githubusercontent.com/20567891/101361804-6dbc1200-38da-11eb-8fb4-581f9a8940cf.png)
```
[0x001e9598]> icq~MethodEnum
[0x001e9598]> icq~+Method
0x0012a95c [0x0061958f - 0x00619598] Landroid/databinding/BindingMethod Ljava/lang/Object;
0x0012a97c [0x006195a0 - 0x006195a0] Landroid/databinding/BindingMethods Ljava/lang/Object;
0x00130bfc [0x001f1f04 - 0x001f272a] Lcom/alibaba/wireless/security/framework/utils/UserTrackMethodJniBridge Ljava/lang/Object;
0x001353fc [0x0026eab8 - 0x0026ec84] Lcom/taobao/android/trade/boost/annotations/MtopParams$RequestMethod Ljava/lang/Enum;
0x001464dc [0x002c76f8 - 0x002c775e] Lcom/taobao/tao/log/godeye/methodtrace/MethodTraceInitializer Ljava/lang/Object;
0x001464fc [0x002c7770 - 0x002c781c] Lcom/taobao/tao/log/godeye/methodtrace/a$1 Ltb/lax$a;
0x0014651c [0x002c782c - 0x002c78d8] Lcom/taobao/tao/log/godeye/methodtrace/a$2 Ltb/lax$a;
0x0014653c [0x002c78e8 - 0x002c7f8c] Lcom/taobao/tao/log/godeye/methodtrace/a Ltb/lav;
[0x001e9598]> 
```

**Comment by trufae** [comment]
Afaik this only works qith the unzipped directory. But its been an experimental feature for a long time, its useful for some situations but not many people payed attention to it.

**Comment by BurningTeng** [comment]
Thanks. For r2frida, the comamnd works well.

![burning](https://user-images.githubusercontent.com/20567891/101442158-abaa4c00-3955-11eb-9f38-56771f0234d3.png)
